bin/lint-staged.js

```diff
@@ -1,36 +1,26 @@
 #!/usr/bin/env node
 
-'use strict'
+import cmdline from 'commander'
+import debug from 'debug'
+import fs from 'fs'
+import supportsColor from 'supports-color'
 
-const fs = require('fs')
+import lintStaged from '../lib/index.js'
+import { CONFIG_STDIN_ERROR } from '../lib/messages.js'
 
 // Force colors for packages that depend on https://www.npmjs.com/package/supports-color
-const supportsColor = require('supports-color')
 if (supportsColor.stdout) {
   process.env.FORCE_COLOR = supportsColor.stdout.level.toString()
 }
 
 // Do not terminate main Listr process on SIGINT
 process.on('SIGINT', () => {})
 
-const pkg = require('../package.json')
-require('please-upgrade-node')(
-  Object.assign({}, pkg, {
-    engines: {
-      node: '>=12.13.0', // First LTS release of 'Erbium'
-    },
-  })
-)
```

Comment:
I removed `please-upgrade-node` and opted to simply define the Node.js versions in the `engines` field of the _package.json_.